### ISSUE TITLE
Removal of expired DCC

### ISSUE THREAD

**Issue by ryanbnl** [ehn-dcc-development/eu-dcc-overview#21]
How should we handle this?

**Comment by francoiskarman** [comment]
Can we agree that, in the described case, the country XX didn't follow the guidance?
The DSC should have a **pkup** valid from/to the 2022-01-01 and 2022-03-31 respectively and a certificate valid from/to the 2022-01-01 and 202**3**-03-31.

As a result, the issue would have to be dealt with by the country XX, republishing the associated DCC with a new and correctly defined DSC.

**Comment by ryanbnl** [comment]
Yes I totally agree - the country did not follow the guidance in that case.

**Comment by ryanbnl** [comment]
In practise there is a second use-case for the DCC over and above travel - that of continuous care. By providing a statement on the holder's medical history, cryptographically signed by the Health Authority where the treatment occurred it has value even after the nominal expiry. Specifically as part of the patient's history.

Although the DCC is not explicitly designed for this case, it will and in fact is being used for that case.

For this scenario there is a need to keep the cryptographic materials after their nominal expiry date.
